components/FeedbackCTA.js

```diff
@@ -4,13 +4,11 @@ import styles from '../styles/FeedbackCTA.module.css'
 
 export default function FeedbackCTA({id}) {
   return (
-    <Link href={`/planning-applications/${id}/feedback`}>
-      <a className={styles.button}>
-        We want your feedback!
-        <div className={styles.arrow}>
-          <Image src="/icons/arrow-right-yellow.svg" alt="" width={14} height={20} />
-        </div>
-      </a>
-    </Link>
-  )
+    (<a href={`/planning-applications/${id}/feedback`} className={styles.button} passhref="true">
```

Comment:
Should this `<a>` tag be a Link instead?

Reply:
I believe this was an auto fix applied by Next when upgrading to V13, some changes made to the Link component. Happy to revert it however, as this change seems to work either way